addons/mail/static/src/core/common/composer.xml

```diff
@@ -88,7 +88,7 @@
                 </div>
                 <div t-if="extended and !props.composer.message" class="d-flex align-items-center mt-2 gap-2">
                     <t t-call="mail.Composer.sendButton"/>
-                    <span t-if="!isSendButtonDisabled and !props.composer.message" t-out="SEND_KEYBIND_TO_SEND"/>
+                    <span t-if="!isSendButtonDisabled and !props.composer.message" class="d-none d-md-block" t-out="SEND_KEYBIND_TO_SEND"/>
```

Comment:
We prefer not showing in DOM at all, as to not pollute DOM with things that are unused.
I'd suggest to add `isMobileOS()`, as mobile OS don't have keyboard thus showing keybind is irrelevant for them.
Small size computers should still show the bind, even when it takes half of screen horizontally.